plugins/woocommerce/src/Blocks/BlockTypes/ProductFilters.php

```diff
@@ -18,7 +18,7 @@ class ProductFilters extends AbstractBlock {
 	 * @return string[]
 	 */
 	protected function get_block_type_uses_context() {
-		return array( 'postId', 'query', 'queryId' );
+		return array( 'postId', 'query', 'queryId', 'woocommerce/product-collection' );
```

Comment:
The PC block doesn't provide this context (`woocommerce/product-collection`), so why do we need this change?

You can use `productCollectionLocation` without declaring anything, as that context is injected into the block that uses the `query` context.

Reply:
Ah sorry I was testing something and accidentally left it there. I pushed update to remove it. https://github.com/woocommerce/woocommerce/pull/56038/commits/6794aa46348f605c5197c299bdeb999a6f1540b0